packages/cli/config-management/docs-configuration/src/DocsConfiguration.ts

```diff
@@ -1,11 +1,12 @@
 import { Audiences } from "@fern-api/config-management-commons";
 import { AbsoluteFilePath } from "@fern-api/fs-utils";
-import { FernRegistry } from "@fern-fern/registry-node";
+import type { FernRegistry } from "@fern-fern/registry-node";
 
 export interface DocsConfiguration {
     navigation: DocsNavigationConfiguration;
     title: string | undefined;
     logo: ImageReference | undefined;
+    logoHref: FernRegistry.docs.v1.write.Url | undefined;
```

Comment:
The way this is modeled it's possible to have an invalid state:

```ts
logo: undefined,
logoHref: "https://logo.com/image.png"
```

^ this is invalid because you can't have an href without a logo image.

you can do:

```ts
logo: Logo | undefined;

interface Logo {
  reference: ImageReference;
  href: FernRegistry.docs.v1.write.Url | undefined;
}
```

ideally we would do the same thing in fdr but nbd

Reply:
yeah i said not to cause it would make it a much heavier backend change (would need to introduce logoV2 and convert)

Reply:
yeah figured